frontend/pkg/frontend/frontend.go

```diff
@@ -36,14 +36,15 @@ const (
 )
 
 type Frontend struct {
+	conn     *sdk.Connection
 	logger   *slog.Logger
 	listener net.Listener
 	server   http.Server
 	cache    Cache
-	dbClient DBClient
+	DbClient database.DBClient
```

Comment:
nit: Any reason why this is being made public?  I see in **cmd.go** it's used to call `DBConnectionTest` but that could also be done with the local `dbClient` variable being passed to `NewFrontend`.  That's the only place I see it being externally accessed at the moment.

Reply:
Ah it's used in all of the `ArmResource*` functions as well - like anytime we need to retrieve or set something in the database.

Reply:
Yeah but those are all methods of `Frontend`, so they have access to private members.

Reply:
Ah good point, I can make that change